### PR TITLE
renamed plant partial for index rendering

### PR DESCRIPTION
**Pull request by uzorjchibuzor**

### CODE REVIEW

File: app/assets/stylesheets/application.tailwind.css [hunk 1]
```diff
@@ -41,12 +40,12 @@
...
-  .notice { 
-    @apply border border-solid p-5 rounded items-center absolute top-0 h-16 w-fit top-2.5 right-2.5
+  .notice {
+    @apply border border-solid p-5 rounded items-center absolute top-0 h-16 w-fit top-2.5 right-2.5;
   }
 
-  .footer { @apply
-    top-full sticky 
+  .footer {
+    @apply top-full sticky;
```
Comment (czepesch) [inline app/assets/stylesheets/application.tailwind.css:48]: it works even without semicolon, but it's ok

File: app/views/plants/index.html.haml [hunk 2]
```diff
@@ -8,4 +8,4 @@
       = f.search_field :name_or_latin_cont, :class => 'text-field'
       = f.submit 'Search', :class => 'btn-primary'
   
-  = render 'plants/plant_table'
\ No newline at end of file
+  = render 'plant'
```
Comment (czepesch) [inline app/views/plants/index.html.haml:11]: I would make it `= render @plants`, but it's minor

**Review by czepesch — APPROVED** [review]
go

File: app/assets/stylesheets/application.tailwind.css (hunk 1 above)
Comment (uzorjchibuzor) [reply app/assets/stylesheets/application.tailwind.css:48]: That's odd, I want uniformity, I will put it back

**Pull request merged**